packages/runtime/src/useCases/transport/messages/SendMessage.ts

```diff
@@ -73,21 +74,53 @@ export class SendMessageUseCase extends UseCase<SendMessageRequest, MessageDTO>
 
-        if (!_.isEqual(transformedContent.toJSON(), localRequest.content.toJSON())) {
-            return RuntimeErrors.general.invalidPropertyValue("The sent Request must have the same content as the LocalRequest.");
-        }
+            if (!_.isEqual(transformedContent.toJSON(), localRequest.content.toJSON())) {
+                return RuntimeErrors.general.invalidPropertyValue("The sent Request must have the same content as the LocalRequest.");
+            }
+
+            if (recipients.length > 1) return RuntimeErrors.general.invalidPropertyValue("Only one recipient is allowed for sending Requests.");
 
-        if (recipients.length > 1) return RuntimeErrors.general.invalidPropertyValue("Only one recipient is allowed for sending Requests.");
+            const recipient = CoreAddress.from(recipients[0]);
+            if (!recipient.equals(localRequest.peer)) return RuntimeErrors.general.invalidPropertyValue("The recipient does not match the Request's peer.");
+            return;
+        }
 
-        const recipient = CoreAddress.from(recipients[0]);
-        if (!recipient.equals(localRequest.peer)) return RuntimeErrors.general.invalidPropertyValue("The recipient does not match the Request's peer.");
+        const peerInDeletionAddressArray: string[] = [];
+        const missingOrInactiveRelationshipAddressArray: string[] = [];
+        const recipientsCoreAddress = recipients.map((r) => CoreAddress.from(r));
+        for (const recipient of recipientsCoreAddress) {
+            const relationship = await this.relationshipsController.getActiveRelationshipToIdentity(recipient);
+            if (!relationship) {
+                missingOrInactiveRelationshipAddressArray.push(recipient.address);
+            }
+            if (relationship?.peerDeletionInfo?.deletionStatus) {
+                peerInDeletionAddressArray.push(recipient.address);
+            }
+        }
+        if (missingOrInactiveRelationshipAddressArray.length > 0) {
+            if (!missingOrInactiveRelationshipAddressArray[1]) {
+                return TransportCoreErrors.messages.missingOrInactiveRelationship(missingOrInactiveRelationshipAddressArray[0]);
+            }
+            return TransportCoreErrors.messages.missingOrInactiveRelationships(missingOrInactiveRelationshipAddressArray);
+        }
+        if (peerInDeletionAddressArray.length > 0) {
+            if (!peerInDeletionAddressArray[1]) {
+                return TransportCoreErrors.messages.peerInDeletion(
+                    `The recipient with the address '${peerInDeletionAddressArray[0]}' has an active IdentityDeletionProcess, so you cannot send them a Message.`
+                );
+            }
+            return TransportCoreErrors.messages.peerInDeletion(
+                `The recipients with the following addresses '${peerInDeletionAddressArray}' have an active IdentityDeletionProcess, so you cannot send them a Message.`
+            );
```

Comment:
It would be more consequent to return both the peers in deletion and the peers with an inactive relationship in the error message.